lisa/sut_orchestrator/azure/tools.py

```diff
@@ -225,9 +225,9 @@ def get_pool_records(self, pool_id: int, force_run: bool = False) -> Dict[str, s
         result = self.run(
             str(pool_id),
             force_run=force_run,
-            expected_exit_code=0,
-            expected_exit_code_failure_message="failed to get pool",
         )
+        # some distro return 4, for example, Ubuntu Server 1804
+        assert_that(result.exit_code).described_as("failed to get pool").is_in(0, 4)
```

Comment:
we can add one more comment after figuring out what the error code 4 stands for.

Reply:
I don't know what code 4 is here, and it succeeds. So I just record the distro name here.